docs/README.md

```diff
@@ -0,0 +1,55 @@
+# Building the docs
```

Comment:
IMO this README should be removed and added [here](https://github.com/CQCL/pytket-docs-theming/tree/main/extensions_deps). A link should then be added to the extensions repository.

This means the instructions can be updated in once place if necessary.

Reply:
Removed, find the instructions here instead -> https://github.com/CQCL/pytket-docs-theming/tree/main/extensions#building-the-api-docs-for-pytket-extensions

Reply:
Sounds good to me!